docs/docs/how-to/performance/improving-site-performance.md

```diff
@@ -81,6 +81,14 @@ There are a number of places to put an inlined script, depending whether you nee
 
 Note that if you are already using [html.js](/docs/custom-html/), you should modify that file to include your snippet instead of using `onPreRenderHTML`. This will have the same behavior.
 
+#### Step 3: Consider using tools that mitigate third-party script impact
+
+Finally, you may want to try new tools that are emerging to mitigate the impact of scripts on web performance.
+
+One option is [Partytown](https://github.com/BuilderIO/partytown), a lazy-loaded library to help relocate resource intensive scripts into a web worker, and off of the main thread.
```

Comment:
I would remove this for now - we're going to integrate this in Gatsby itself and it's need some setup to get this right so bluntly putting it in our docs doesn't feel right